kitsune/users/views.py

```diff
@@ -187,15 +187,51 @@ def deactivation_log(request):
     return render(request, "users/deactivation_log.html", {"deactivations": deactivations})
 
 
+def questions(request, username):
+    # plus sign (+) is converted to space
```

Comment:
The 3 views here basically they are doing the same thing. Personally I would look into combining them into a single one and maybe add a url with optional params

Reply:
I think that would add needless complexity: the template logic for documents is very different, and if we ever invest in rendering questions/answers in a better way than just a list of titles, their template logic is likely to be different too.

Additionally, the documents/answers views cater towards users who are contributors, whereas the questions view is for users who aren't, and I imagine combining them all into one contributions route will confuse non-contributors.

Reply:
Actually a decorator DRYing the user_profile fetching logic works nicely here